drivers/net/wireless/realtek/rtl8192cu/hal/rtl8192c/rtl8192c_phycfg.c

```diff
@@ -2360,9 +2360,6 @@ phy_TxPwrIdxToDbm(
 		Offset = -7;
 		break;
 
-	case WIRELESS_MODE_G:
-	case WIRELESS_MODE_N_24G:
-		Offset = -8;
```

Comment:
The original code is showing that WIRELESS_MODE_G and WIRELESS_MODE_N_24G have been handled explicitly and need ``Offset = -8``. The default case generally means we'll pick a sane value, but it's less clear if it will be correct if new modes are added.

I think the real bug is the missing break statement after the assignment, and fixing it that way makes it clear that WIRELESS_MODE_G/WIRELESS_MODE_N_24G are being handled explicitly.

The compiler will generate the same code is all cases, but add the break rather than removing the labels provides more information to a programmer reading the code.

This also applies to the following two similar patches.

Reply:
Will push that change here shortly.